scripts/build_and_install.sh

```
@@ -69,29 +75,29 @@
         -DCCAP_BUILD_EXAMPLES=OFF \
         -DCCAP_BUILD_TESTS=OFF \
         -DCCAP_INSTALL=ON
-    
+
     # Build
     echo "Building $config..."
     cmake --build . --config "$config" --parallel $(detectCores 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)